tests/post/steps/test_volume.py

```diff
@@ -117,6 +151,34 @@ def _check_size():
         name='checking size of PersistentVolume {}'.format(name)
     )
 
+
+@then(parsers.parse("the Volume '{name}' does not exists"))
+def check_volume_absent(host, name, k8s_custom_client, k8s_client):
+    def _check_volume():
+        volume = _get_volume(k8s_custom_client, name)
+        if volume is not None:
+            assert False, 'Volume {} still exist'.format(name)
+    utils.retry(
+        _check_volume, times=30, wait=2,
+        name='checking the absence of volume {}'.format(name)
+    )
+
+
+@then(parsers.parse("the PersistentVolume '{name}' does not exists"))
+def check_pv_absent(name, k8s_client):
+    def _check_pv():
+        try:
+            pv = k8s_client.read_persistent_volume(name)
+        except (ApiException, HTTPError) as exc:
+            if isinstance(exc, ApiException) and exc.status == 404:
+                return None
+        if pv is not None:
+            assert False, 'PersistentVolume {} exist'.format(name)
+    utils.retry(
+        _check_pv, times=30, wait=2,
```

Comment:
As before, I think we can decrease the retry parameters: `times=10, wait=1` or `times=10, wait=2`.